src/java.base/share/classes/java/lang/ref/ReferenceQueue.java

```diff
@@ -173,6 +180,12 @@ boolean enqueue(Reference<? extends T> r) { /* Called only by Reference class */
      * available without further delay then it is removed from the queue and
      * returned.  Otherwise this method immediately returns {@code null}.
      *
+     * @apiNote
+     * If the returned reference was added to this queue by a call to
+     * {@link Reference#enqueue()} instead of by the garbage collector, its
+     * former referent (which has since been cleared) could still be strongly
+     * reachable.
```

Comment:
See my comments on the corresponding Reference.enqueue() apiNote. I'm not sure we want to have this issue sprinkled around in the apiNotes for several methods.

If we do want to address this issue -- and it's not clear to me that we do -- then perhaps we should do it in a central location?

Reply:
We can cover this issue just in `Reference.enqueue()`. I'll take out these `@apiNote`s, and put in `@see Reference.enqueue()`.